NMS-13694: Migrate Discourse charts article into main docs (#4126)
OpenNMS/opennms: docs/modules/operation/pages/admin/webui/dashboard.adoc

= Dashboard

> In Network Operations Centers NOC an overview about issues in the network is important and often described as dashboards.

Reviewer: ```suggestion
In Network Operations Centers (NOCs) an overview of issues in the network is important and often presented as dashboards.
```
